Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -1675,10 +1675,14 @@ private void ChangePhaseOfActionInternal(int actionIndex, TriggerState* actionSt
 
             // Update action state.
             var newState = trigger;
-            newState.flags = actionState->flags; // Preserve flags.
-            newState.phase = newPhase;
+
+            // We need to update the magnitude before we copy the flags across as the copied flags may contain
+            // the `HaveMagnitude` flag which would give us an incorrect magnitude value. (case 1239551)
             if (!newState.haveMagnitude)
```

Comment:
@Rene-Damm This change is causing several tests to fail.
They all have messages like this
_InvalidOperationException: Cannot query value of control '/VirtualMouse/scroll' before 'VirtualMouse' has been added to system!'_

Should I check if they have been added to the system before calling ComputeMagnitude? How can i do this?